src/views/HomePage.vue

```diff
@@ -17,6 +17,26 @@
             @select-course="onSelectCourse">
         </course-list>
         <course-create @create-course="onCreateCourse"></course-create>
+        <button class="bookmarklet-button"
+          @click="openBookmarkletModal"
+        >
+        Enable NB Sidebar
+        </button>
+        <modal name="bookmarklet-modal" height="auto" width="70%" >
+        <div class="bookmarklet-modal">
+        <p>To enable the NB sidebar for a class reading, either drag (don't click) this <a :href = "bookmarklet">NB Bookmarklet</a> link to your bookmarks bar, or make a new bookmark with the code below as the URL.</p>
+    <p>When on the class reading page, just click on the bookmark and the sidebar should appear.</p>
+    <pre>
+      <code>
+        javascript:(function(){
+          let s = document.createElement('script');
+          s.src= '{{pathname}}/js/bundle.js';
```

Comment:
the path name for the client name should be `{{pathname}}/client/js/bundle.js` since in production we build the client and transferring it to the client directory.